sku.routes.js

```diff
@@ -1,15 +1,18 @@
 const fs = require('fs');
 const path = require('path');
-
-const undocumentedExports = ['useThemeName', 'BoxRenderer'];
```

Comment:
Oh nice—this made me realise that the `useThemeName` route is a 404 if you try to hit it directly. Great clean up.